R/rules-line-breaks.R

```diff
@@ -447,3 +447,14 @@ remove_empty_lines_after_opening_and_before_closing_braces <- function(pd) {
 
   pd
 }
+
+
+reduce_extra_blank_lines_between_scopes <- function(pd, allowed_blank_lines = 2L) {
+  # Calculate the maximum allowed lag_newlines
```

Comment:
sory to be so particular but can you 
```
  pd$lag_newlines <- pmin(pd$lag_newlines, allowed_blank_lines + 1L)
```
and remove the comments? If you look at other transformers (i.e. the rule functions), they rarely contain comments, except where something truly _weird_ needs explanation.

Reply:
That's quite clean. Thanks!